synapseAnnotations/data/experimentalData.json

```diff
@@ -307,6 +307,11 @@
         "value": "bone marrow",
         "description": "The soft, fatty, vascular tissue that fills most bone cavities and is the source of red blood cells and many white blood cells.",
         "source": "http://purl.obolibrary.org/obo/BTO_0000141"
+      },
+      {
+        "value": "skin",
```

Comment:
We already have skin here: https://github.com/Sage-Bionetworks/synapseAnnotations/blob/534c82b81d33562ea8d6bfda9cc1c3b2dfa621b8/synapseAnnotations/data/experimentalData.json#L251-L255

Is that definition not sufficient?